.github/workflows/smoketests.yml

```diff
@@ -0,0 +1,29 @@
+name: Smoke Tests
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  trigger-smoketests:
+    name: Trigger CLI Smoke Tests
+    runs-on: ubuntu-latest
+    concurrency:
+      group: smoketests-${{ github.ref }}
+      cancel-in-progress: true
+
+    steps:
+      - name: Trigger smoke tests in DefangLabs/defang-mvp
```

Comment:
FYI, elsewhere in this repo we use an action for this: https://github.com/DefangLabs/defang/blob/main/.github/workflows/go.yml#L355-L361